packages/fxa-payments-server/src/lib/hooks.test.tsx

```diff
@@ -209,8 +209,8 @@ describe('useInfoBoxMessage', () => {
     const couponDurationDate = getByTestId('couponDurationDate').textContent;
     expect(messageText).toBe(CouponInfoBoxMessageType.Repeating);
     expect(couponDurationDate).not.toBeNull();
-    expect(couponDurationDate?.slice(0, 8)).toBe(
-      expectedCouponDurationDate.slice(0, 8)
+    expect(couponDurationDate?.slice(0, 7)).toBe(
```

Comment:
Why is this needed? Should this be a separate fix?

Reply:
Looks like this is from https://github.com/mozilla/fxa/pull/14069/files#diff-0d3612ed12d37aac060862c13f9face9d622f87013ed430f4c09fe83077b9ba5?